ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -118,6 +118,24 @@
   @Default("50")
   public final int storeMinUsedCapacityToTriggerCompactionInPercentage;
 
+  /**
+   * The number of buckets for stats bucketing, a value of 0 will disable bucketing.
+   */
+  @Config("store.stats.bucket.count")
+  @Default("24")
+  public final int storeStatsBucketCount;
+
+  /**
+   * The time coverage of each buckets in milliseconds.
+   */
+  @Config("store.stats.bucket.span.in.ms")
+  @Default("3600000")
```

Comment:
Does bucket span need to be in ms? why not minutes? What is the minimum granularity are we willing to offer